app/[locale]/admin/categories/page.jsx

```
@@ -0,0 +1,680 @@
+            <div>
+              <Label htmlFor="category-name">Category Name</Label>
+              <Input
+                id="category-name"
+                placeholder="e.g., Islamic Philosophy"
+                value={formData.name}
+                onChange={(e) => {
+                  setFormData({ ...formData, name: e.target.value });
+                  setFormError("");
+                }}
+                className="mt-1"
+                autoFocus
```

ESLint

[error] The autoFocus prop should not be used, as it can reduce usability and accessibility for users.

(jsx-a11y/no-autofocus)